data/product.d/ovirt.conf

```diff
@@ -4,25 +4,28 @@
 product_name = oVirt Node Next
 
 [Base Product]
-product_name = CentOS Linux
+product_name = CentOS Stream
```

Comment:
This change breaks the `rhel_products_test` unit test. It is necessary to remove `centos.conf` from `["rhel.conf", "centos-stream.conf", "centos.conf", "ovirt.conf"]`.

Reply:
done